### PR TITLE
[Chore] Update code paths in saltcorn-cli/README.md

### PR DESCRIPTION
**Pull request by Atozzio**
Hello Tom! I hope you are doing fine. It's wonderful to see the increased activity in Saltcorn community.
I'm currently trying to get familiar with the whole project structure and hope that I may be able to take a few tickets.
However, when I was troubleshooting the deployment steps on my local machine, I came across this issue when I try to reach the corresponding code through readme, the project structure may have changed since it was initially added. Nevertheless, I probably should not update this file manually, considering we may have a small tool for updating all the package versions across all saltcorn files, and it is better to point the PR to a dev branch other than master.

Best regards,
Zihao

### CODE REVIEW

**Comment by glutamate** [comment]
Hello @Atozzio! Good to hear form you! sorry that I misses this PR, will look now. Hope all is well with you!

File: packages/saltcorn-cli/README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ OPTIONS
   -f, --force  force command execution
 ```
 
-_See code: [src/commands/add-schema.js](https://github.com/saltcorn/saltcorn/blob/v0.9.6-beta.0/src/commands/add-schema.js)_
+_See code: [src/commands/add-schema.js](https://github.com/saltcorn/saltcorn/blob/v0.9.6-beta.0/packages/saltcorn-cli/src/commands/add-schema.js)_
```
Comment (glutamate) [inline packages/saltcorn-cli/README.md:92]: unfortunately these are created automatically on release by oclif and would be overwritten. I think we need to fix the ocliff configuration instead

**Comment by Atozzio** [comment]
@glutamate Apologies! I just got back from vacation.
I have update the oclif config and tested the urls by running the "prepack" command. It should work this time!

Also, I am planning to pick some random tickets from the issues list, are there any urgent ones or some tasks in the roadmap that seems to be beginner-friendly? I can start with them.

**Comment by Atozzio** [comment]
Do you mind rerun the failed job? The failed pytest case has passed locally, and we did not change any 2FA stuff in this PR.

**Comment by glutamate** [comment]
Thank you, merged. @Atozzio are you still interested in security? We could talk about doing some interesting projects in there, like getting scanners into CI and improving the python security test suite

**Pull request merged**